src/components/SecondPage/SecondPageModal/EditModal.js

```diff
@@ -0,0 +1,105 @@
+import React from "react";
+import history from "../../../history";
+import SecondModal from "./SecondModal";
+import SecondPage from "../SecondPage";
+import Backdrop from "../../Backdrop/Backdrop";
+import "./modal.css";
+
+
+class EditModal extends React.Component {
```

Comment:
Can you break down this component?
I can see multiple components. defined inside it.

Don't be afraid to create multiple files for a component just makes sure the component name reflects what the compone t does

eg.

```
const title = () => {
      return (
        <div className="title-box">
          <label>
            <h5>Title</h5>
          </label>
          <input placeholder="Learn a new programming language" />
        </div>
      );
    };
```

Will be a file called 
EditModalTitle 
exporting the component 

And aI can. use it inside the Editmodal